package.json

```diff
@@ -147,8 +147,10 @@
     },
     "collectCoverage": true,
     "collectCoverageFrom": [
-      "**/*.{js,vue}",
-      "!**/node_modules/**"
+      "**/*.{js,vue}"
+    ],
+    "coveragePathIgnorePatterns": [
+      "/node_modules/"
```

Comment:
In practice I found that the `collectCoverageFrom` directive intended to prevent looking at `node_modules` didn't work; I found this worked better. I may have missed something, but this speeds up the coverage collection quite a bit in my tests.